app/js/views/reminders.jsx

```diff
@@ -159,6 +167,35 @@ export default class Reminders extends React.Component {
       });
   }
 
+  onQuery({ recipients, due }) {
+    if (recipients.length !== 1) {
+      console.error('Query must be have only 1 recipient.');
+    }
+
+    // Reminders last for 3 hours by convention.
+    const reminder = this.state.reminders.find((reminder) => {
+      return reminder.recipients.includes(recipients[0])
+        && reminder.due <= due && due <= (reminder.due + 3 * 60 * 60 * 1000);
+    });
+
+    console.log(reminder);
```

Comment:
Maybe remove this or add some extra context: `` `Reminder: ${reminder}` ``.  You might see just `null` in the console output otherwise